inference-engine/thirdparty/clDNN/api/loop.hpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+#include <vector>
+#include <functional>
+#include "primitive.hpp"
+#include "topology.hpp"
+
+#define DEFAULT_MAX_NUM_ITERATION 128
```

Comment:
Where this max iterations number comes from? Is it defined in ngraph spec? Why exactly 128?